src/dymaptic.GeoBlazor.Core/Components/Layers/FeatureLayer.cs

```diff
@@ -879,23 +880,20 @@ public async Task<int> QueryFeatureCount(Query? query = null, CancellationToken
     public async Task<FeatureSet?> QueryFeatures(Query? query = null, CancellationToken cancellationToken = default)
     {
         IJSObjectReference abortSignal = await AbortManager!.CreateAbortSignal(cancellationToken);
+        Guid queryId = Guid.NewGuid();
 
-        FeatureSet? result = await JsLayerReference!.InvokeAsync<FeatureSet?>("queryFeatures", cancellationToken,
-            query, new { signal = abortSignal }, DotNetObjectReference.Create(this), View?.Id);
+        FeatureSet result = (await JsLayerReference!.InvokeAsync<FeatureSet?>("queryFeatures", cancellationToken,
```

Comment:
are we expecting each `result` to not be null?

Reply:
yes, this should always return something, even if the `FeatureSet` is empty of graphics.